### ISSUE TITLE
Can you extend this to also work with embedded content?

### ISSUE THREAD

**Issue by starikcetin** [erdemolcay/unblock-imgur#2]
The extension is working fine with direct visits, but embedded images are still broken for me.

**Comment by erdemolcay** [comment]
It works for embedded content in my tests.

Example: https://meta.stackexchange.com/questions/75491/how-to-upload-an-image-to-a-post

If your problem still exists, feel free to open a new issue with example URLs.
